src/types/onyx/Policy.ts

```diff
@@ -68,18 +69,101 @@ type TaxRatesWithDefault = {
     taxes: TaxRates;
 };
 
-// These types are for the Integration connections for a policy (eg. Quickbooks, Xero, etc).
-// This data is not yet used in the codebase which is why it is given a very generic type, but the data is being put into Onyx for future use.
-// Once the data is being used, these types should be defined appropriately.
-type ConnectionLastSync = Record<string, unknown>;
-type ConnectionData = Record<string, unknown>;
-type ConnectionConfig = Record<string, unknown>;
+type ConnectionLastSync = {
+    successfulDate?: string;
+    errorDate?: string;
+    isSuccessful: boolean;
+    source: 'DIRECT' | 'EXPENSIFYWEB' | 'EXPENSIFYAPI' | 'AUTOSYNC' | 'AUTOAPPROVE';
+};
+
+type Account = {
+    glCode?: string;
+    name: string;
+    currency: string;
+    id: number;
+};
+
+type Employee = {
+    id: number;
+    firstName: string;
+    lastName: string;
+    name: string;
+    email: string;
+};
+
+type Vendor = {
+    id: number;
+    name: string;
+    currency: string;
+    email: string;
+};
+
+type ConnectionData = {
+    country: string;
+    edition: string;
+    homeCurrency: string;
+    isMultiCurrencyEnabled: boolean;
+
+    journalEntryAccounts: Account[];
+    bankAccounts: Account[];
+    creditCards: Account[];
+    accountsReceivable: Account[];
+    accountsPayable: Account[];
+    otherCurrentAssetAccounts: Account[];
+
+    taxCodes: string[];
```

Comment:
Here's an example of this array showing the format:

```js
[
    {
        "totalTaxRateVal": "0.00%",
        "simpleName": "20.0% ECG",
        "taxCodeRef": "7",
        "taxRateRefs": {
            "6": "20.00",
            "13": "-20.00"
        },
        "name": "EC Goods Standard"
    },
    {
        "totalTaxRateVal": "20.00%",
        "simpleName": "20.0% S",
        "taxCodeRef": "2",
        "taxRateRefs": {
            "1": "20.00"
        },
        "name": "Standard"
    }
]
```

Reply:
defined `TaxCode` type

thanks for the sample data 👍